### PR TITLE
add new types for loadable/server 5.12

### PR DESCRIPTION
**Pull request by jupadev**
- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] Add or edit tests to reflect the change. (Run with `npm test`.)
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] Run `npm run lint package-name` (or `tsc` if no `tslint.json` is present).

If changing an existing definition:
- [x] Provide a URL to documentation or source code which provides context for the suggested changes: <<https://loadable-components.com/docs/api-loadable-server/>>
- [x] If this PR brings the type definitions up to date with a new version of the JS library, update the version number in the header.

### CODE REVIEW

File: types/loadable__server/index.d.ts [hunk 1]
```diff
@@ -104,6 +108,21 @@ export class ChunkExtractor {
 	 * Get style links as an array of React `<link>` elements
 	 */
   	getStyleElements(attr?: {} | AttrFn): Array<ReactElement<{}>>;
+
+	/**
+	 * Get inline style links as a string of <link> tags (returns a promise)
+	 */
+  	getInlineStyleTags(attr?: {} | AttrFn): Array<ReactElement<{}>>;
```
Comment (peterblazejewicz) [inline types/loadable__server/index.d.ts]: Should probably read:
``ts
getInlineStyleTags(attr?: {} | AttrFn): Promise<Array<ReactElement<{}>>>;
```

File: types/loadable__server/loadable__server-tests.tsx [hunk 2]
```diff
@@ -104,6 +107,36 @@ const attrFn: AttrFn = (chunk) => {
...
+// getInlineStyleTags
+{
+    // Should return an array of React elements
+    const elements: Array<React.ReactElement<{}>> = getInlineStyleTags();
+    const elementsWithAttrs: Array<React.ReactElement<{}>> = getInlineStyleTags(
+        attributes
+    );
+    const elementsWithAttrFn: Array<React.ReactElement<{}>> = getInlineStyleTags(
+        attrFn
+    );
```
Comment (peterblazejewicz) [inline types/loadable__server/loadable__server-tests.tsx]: As this should return a promise, those details should reflect this, followign library:
https://github.com/gregberge/loadable-components/blob/f2a6bbd6d7e96780eed8422011942f2f2785c96f/packages/server/src/ChunkExtractor.test.js#L367
https://github.com/gregberge/loadable-components/blob/b0a47face8f84545a3069930ddc2ec572c683d45/website/src/pages/docs/api-loadable-server.mdx#L142-L145

Comment (jupadev) [reply types/loadable__server/loadable__server-tests.tsx]: f1dc2df

File: types/loadable__server/index.d.ts (hunk 1 above)
Comment (jupadev) [reply types/loadable__server/index.d.ts]: thanks @peterblazejewicz 32210f1

**Review by peterblazejewicz — APPROVED** [review]
@jupadev thx for the changes, for me LGTM!

**Pull request merged**